helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ClusterStatusMonitor.java

```diff
@@ -881,7 +910,7 @@ public String clusterBeanName() {
    * @param instanceName
    * @return instance bean name
    */
-  private String getInstanceBeanName(String instanceName) {
+  public String getInstanceBeanName(String instanceName) {
```

Comment:
I don't have strong preference here.
- Shall protected or package-private enough here?
- Or, on the other hand, with these two methods made to the public, we can avoid several test codes using INSTANCE_DN_KEY and RESOURCE_DN_KEY directly. Not a must in this PR, though.

Reply:
I changed all getXXXBeanName() methods to protected.